tests/none-backend.nix

```diff
@@ -80,9 +93,15 @@ makeTest {
   nodes =
     { coordinator =
         { config, pkgs, ... }:
-        { environment.systemPackages =
-            [ nixops pkgs.stdenv pkgs.vim pkgs.apacheHttpd pkgs.busybox
-              pkgs.module_init_tools pkgs.perlPackages.ArchiveCpio ];
+        { environment.systemPackages = [ nixops ];
+          # This is needed to make sure the coordinator can build the
+          # deployment without network availability.
+          environment.etc.nix-references.source = let
```

Comment:
Be sure to also cherry-pick 370e9284c7241fe8a66b33e98bda7c0433159382, which does the same for the Hetzner tests.